### DISCUSSION TITLE
Does not work after the update

### DISCUSSION THREAD

**Discussion by Gor80hd** [d8ahazard/sd_dreambooth_extension#766] [Q&A]
The new version of Dreambooth does not work at all on 8gb video cards. I have RTX 3070 and 16GB of RAM, and when the training starts, I immediately get the error "No executable batch size found, reached zero.".
The memory of the video card is not fully filled when an error appears. I use xformers and LoRA to reduce the load, but no results. And this problem is that on Stable Diffusion 1.5, that on 2.1 or 2.0. I absolutely do not know how to deal with it.

**Comment by aearone** [comment]
Same for me. It looks like we need to wait for a fix or a return to an older version. I also don't like the simplification of setup in the new version of DreamBooth at all, as there are far fewer settings options and there are no new setup instructions yet.

**Comment by mykeehu** [comment]
The error [I reported here](https://github.com/d8ahazard/sd_dreambooth_extension/issues/760) is caused by two things: generating the preview image, and/or using class images.

**Reply by Gor80hd** [reply]
unfortunately it didn't help me

**Comment by lolxdmainkaisemaanlu** [comment]
for now I'm training LoRA on a GTX 1060 6GB with this repo - https://github.com/bmaltais/kohya_ss. working just fine

**Reply by aearone** [reply]
> for now I'm training LoRA on a GTX 1060 6GB with this repo - https://github.com/bmaltais/kohya_ss. working just fine

Are there detailed and up-to-date installation instructions for LoRA on stable diffusion from AUTOMATIC1111?

**Comment by healthyfat** [comment]
Try older commit. This one is recommended and it works for me: 

https://github.com/d8ahazard/sd_dreambooth_extension/tree/c5cb58328c555ac27679422b1da940a9b19de6f2

Download zip file and replace your "sd_dreambooth_extension" folder.

**Reply by healthyfat** [reply]
Can't find this commit anywhere. How old is it? The one I provided is from 22nd December.

**Reply by mykeehu** [reply]
Ok, sorry, I missed that, I was looking at the webUI build. I'll delete the comment.

**Comment by allusernamestakenexceptthis** [comment]
does anyone know what caused this to fail? I want to dig deeper in code to try to fix it. but I appreciate it if someone can give me a head start at what to look at.  especially @d8ahazard could know.  thanks
